graphql/schema.graphql

```diff
@@ -532,11 +532,17 @@ type ChildPatchAlias {
 type PatchTriggerAlias {
   alias: String!
   childProject: String!
-  taskSpecifiers: [TaskSpecifier]
-  status: String
-  parentAsModule: String
+  variantsTasks: [VariantTask]!
 }
 
+type PatchTriggerDefinition {
```

Comment:
I don't love that we have two different data structures to represent the same thing, just different fields. Can we combine these into one and just populate the fields we need? (only populate variant tasks for this patch resolver?)